src/elements-experimental/seat-reservation/seat-reservation-place-control/seat-reservation-place-control.scss

```
@@ -118,7 +118,7 @@ $disabled: '[disabled], :disabled, .sbb-reservation-place-control--disabled';
     inset-block-start: 50%;
     transform-origin: center;
     transform: translate(-50%, -50%)
-      rotate(calc(var(--sbb-reservation-place-control-text-rotation) * 1deg));
+      rotateZ(calc(var(--sbb-reservation-place-control-text-rotation) * 1deg));
```

maybe consider adding the unit into the var itslef?
```suggestion
      rotate: calc(var(--sbb-reservation-place-control-text-rotation) * 1deg);
```

---

we have a subtask addressing this kind of "code smell"